src/utils/global/tooltip/Tooltip.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+
+import styles from "./tooltip.module.scss";
+
+interface TooltipProps {
+  text: string;
+  children: React.ReactNode;
+  isShown: boolean;
+  isOption?: boolean;
+  offsetLeft?: number;
+  offsetTop?: number;
+  width?: number;
+}
+
+const Tooltip: React.FC<TooltipProps> = (props) => {
+  const tooltipStyle: React.CSSProperties = {
+    opacity: props.isShown ? 1 : 0,
+    width: `${props.width}rem` || "auto",
```

Comment:
This doesn't seem quite right? `"undefinedrem"` is going to be truthy, so `|| "auto"` will never apply.